test/gurka/test_pbf_api.cc

```diff
@@ -0,0 +1,100 @@
+#include "gurka.h"
```

Comment:
this is a new test that lets us do some basic api testing, specifically using the pbf api